src/types.rs

```diff
@@ -749,6 +740,19 @@ impl<T: ToRedisArgs> ToRedisArgs for Option<T> {
     }
 }
 
+impl <T: ToRedisArgs> ToRedisArgs for &T {
+    fn write_redis_args<W>(&self, out: &mut W)
+    where
+        W: ?Sized + RedisWrite,
+    {
+        ToRedisArgs::write_redis_args(*self, out)
+    }
+
+    fn is_single_arg(&self) -> bool {
+        ToRedisArgs::is_single_arg(*self)
+    }
```

Comment:
I believe this should forward all the other methods as well as `&T` and `T` may behave differently otherwise if `T` has specialized one of the inner methods.

Reply:
Thanks for pointing this out. I left the doc(hidden) stuff to defaults

Reply:
I think those should be forwarded as well

Reply:
If possible